cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/LocalStreamService.java

```diff
@@ -68,10 +68,13 @@ public LocalStreamService(StreamCoordinatorClient streamCoordinatorClient,
 
   @Override
   protected void initialize() throws Exception {
-    for (StreamSpecification streamSpec : streamMetaStore.listStreams(Constants.DEFAULT_NAMESPACE)) {
-      StreamConfig config = streamAdmin.getConfig(streamSpec.getName());
+    String namespace = Constants.DEFAULT_NAMESPACE;
+    //TODO: use streamMetaStore.listStreams() instead
```

Comment:
The StreamService is not specific to one namespace. Its responsible for all streams.